### PR TITLE
Add support for resolving lat lngs.

### PR DESCRIPTION
**Pull request by keyurva**
* Also added support for geoIds.
* The date field is now always output with the name `date`.

### CODE REVIEW

File: simple/stats/importer.py [hunk 1]
```diff
@@ -136,22 +116,41 @@ def remove_pre_resolved(entity: str) -> bool:
             unresolved=unresolved_list,
         )
 
+    def _resolve(self, entity_column_name: str,
+                 entities: list[str]) -> dict[str, str]:
+        if entity_column_name.casefold(
+        ) == constants.INPUT_COLUMN_DCID.casefold():
```
Comment (shifucun) [inline simple/stats/importer.py:122]: very nit, can use a function like check_column(column_name, constatns.xxx) and use casefold() in the function.

File: simple/stats/sample/countries/observations.csv [hunk 2]
```diff
@@ -1,23 +1,14 @@
...
-country/ZWE,var2,2023,76.0
-country/ALB,var2,2023,34.0
-wikidataId/Q22062741,var2,2023,97.0
-country/DZA,var2,2023,92.0
-country/AND,var2,2023,9.0
-country/ASM,var2,2023,34.0
-country/AIA,var2,2023,42.0
-country/WLF,var2,2023,75.0
-country/ESH,var2,2023,65.0
+dcid,date,var1,var2
```
Comment (shifucun) [inline simple/stats/sample/countries/observations.csv:1]: thanks for the update!

File: simple/stats/constants.py [hunk 3]
```diff
@@ -22,9 +22,15 @@
 OBSERVATIONS_FILE_NAME = "observations.csv"
 DEBUG_RESOLVE_FILE_NAME = "debug_resolve.csv"
 
-DCID_OVERRIDE_PREFIX = "dcid:"
+# Input columns.
+INPUT_COLUMN_DCID = "dcid"
+INPUT_COLUMN_GEO_ID = "geoId"
```
Comment (shifucun) [inline simple/stats/constants.py:27]: could the geoId be even more generalized here? like we can do "wikidataId" now.

**Review by shifucun — APPROVED** [review]
very nice to make the input general!

File: simple/stats/importer.py (hunk 1 above)
Comment (keyurva) [reply simple/stats/importer.py]: I've made the logic more generic. PTAL.

File: simple/stats/constants.py (hunk 3 above)
Comment (keyurva) [reply simple/stats/constants.py:27]: Yes, I've made it more generic. PTAL.

File: simple/stats/constants.py [hunk 4]
```diff
@@ -26,6 +26,22 @@
...
+
+e.g. if a column is named "geoId" and the values of that column in the CSV are: 01, 02; 
+then the importer will update the values to geoId/01, geoId/02 before importing them.
+
+Note that the lookup into these columns will always be lower-cased so ensure that 
+the keys in the dictionary are lower-case as well.
+
+Also, keep the keys sorted so it's easier to spot check should the list get large.
+"""
+PRE_RESOLVED_INPUT_COLUMNS_TO_PREFIXES = dict([("dcid", ""),
```
Comment (shifucun) [inline simple/stats/constants.py:41]: for wikidataId, we still want to resolve them to be dcid using the resolve api: 
image

**Pull request merged**